frontend/src/actionCreators/parcelsActionCreator.tsx

```diff
@@ -105,3 +105,18 @@ export const updateParcel = (parcel: IParcel) => (dispatch: Function) => {
     })
     .finally(() => dispatch(hideLoading()));
 };
+
+export const deleteParcel = (parcel: IParcel) => (dispatch: Function) => {
+  dispatch(request(actionTypes.DELETE_PARCEL));
+  dispatch(showLoading());
+  return CustomAxios()
+    .delete(ENVIRONMENT.apiUrl + API.PARCEL_ROOT + `/${parcel.id}`, { data: parcel })
+    .then((response: AxiosResponse) => {
+      dispatch(success(actionTypes.DELETE_PARCEL, response.status));
+      dispatch(hideLoading());
+    })
+    .catch((axiosError: AxiosError) => {
+      dispatch(error(actionTypes.DELETE_PARCEL, axiosError?.response?.status, axiosError));
+    })
+    .finally(() => dispatch(hideLoading()));
```

Comment:
Not that it should be done now, but I think we should plan on moving the loading progress dispatch to the axios interceptor.